src/providers/proxyProvider.ts

```diff
@@ -1,21 +1,24 @@
-import { ProxyProvider } from '@elrondnetwork/erdjs';
+import { ProxyNetworkProvider } from '@elrondnetwork/erdjs-network-providers/out';
 import { Address } from '@elrondnetwork/erdjs/out';
 import { networkSelector } from 'redux/selectors';
 import { store } from 'redux/store';
 import { NetworkType } from 'types';
 
-let proxyProvider: ProxyProvider | null = null;
+let proxyProvider: ProxyNetworkProvider | null = null;
 
 export function initializeProxyProvider(networkConfig?: NetworkType) {
   const initializationNetworkConfig =
     networkConfig || networkSelector(store.getState());
-  proxyProvider = new ProxyProvider(initializationNetworkConfig.apiAddress, {
-    timeout: Number(initializationNetworkConfig.apiTimeout)
-  });
+  proxyProvider = new ProxyNetworkProvider(
+    initializationNetworkConfig.apiAddress,
+    {
```

Comment:
probably should be in the same line, maybe it's a prettier problem?

Reply:
Yes, it's auto-formatted